### PR TITLE
Search page work, 2nd pass

### PR DESCRIPTION
**Pull request by amardesi**

### CODE REVIEW

File: app/jsx/components/ScholWorksComp.jsx [hunk 1]
```diff
@@ -32,73 +45,97 @@ class DotH2 extends React.Component {
...
-    //   if ('pdf' in this.props.result.supp_files && this.props.result.supp_files.pdf !== 0) {
-    //     supp_files.append(<li className="c-scholworks__media-pdf">Contains {this.props.result.supp_files.pdf} additional PDFs</li>);
-    //   }
-    //   if ('audio' in this.props.result.supp_files && this.props.result.supp_files.audio !== 0) {
-    //     supp_files.append(<li className="c-scholworks__media-audio">Contains {this.props.result.supp_files.audio} audio files</li>);
-    //   }
-    // }
-    
+    })
+    console.log(pr.supp_files)
```
Comment (amardesi) [inline app/jsx/components/ScholWorksComp.jsx]: In the middle of working on this right now

File: app/jsx/components/ScholWorksComp.jsx [hunk 2]
```diff
@@ -9,6 +10,18 @@ if (!(typeof document === "undefined")) {
   const dotdotdot = require('jquery.dotdotdot')
 }
 
+class DotAuthorUl extends React.Component {
```
Comment (amardesi) [inline app/jsx/components/ScholWorksComp.jsx]: The authorlist has a particular requirement for the 'et al' link.

File: app/jsx/components/ScholWorksComp.jsx [hunk 3]
```diff
@@ -111,35 +148,36 @@ class ScholWorksComp extends React.Component {
...
             </DotDiv>
           }
           <div className="c-scholworks__media">
             <ul className="c-scholworks__media-list">{ supp_files }</ul>
-            {this.props.result.rights && this.props.result.rights !== 'public' && <img className="c-scholworks__rights" src="/images/cc-by-small.svg" alt="creative commons attribution 4.0 international public license"/>}
+            {pr.rights && pr.rights !== 'public' && <img className="c-scholworks__rights" src="/images/cc-by-small.svg" alt="creative commons attribution 4.0 international public license"/>}
           </div>
         </div>
-        {this.props.result.thumbnail && <img className="c-scholworks__article-preview" src="/images/temp_article.png" alt="article"/>}
+        {pr.thumbnail && <img className="c-scholworks__article-preview" src={"/assets/"+pr.thumbnail.asset_id} width={pr.thumbnail.width} height={pr.thumbnail.height} alt="Article image" />}
```
Comment (amardesi) [inline app/jsx/components/ScholWorksComp.jsx]: Really nice to get this working, thanks for all your hard work Martin.

File: app/jsx/components/SortComp.jsx [hunk 4]
```diff
@@ -41,10 +41,10 @@ class SortComp extends React.Component {
           <select name="rows" id="c-sort2" form="facetForm" value={ this.state.rows } onChange={ this.handleChange }>
           {/* ToDo: Make this more concise */}
           <option value="10">10</option>
-          {(this.props.count > 20) && <option value="20">20</option> }
-          {(this.props.count > 30) && <option value="30">30</option> }
-          {(this.props.count > 40) && <option value="40">40</option> }
-          {(this.props.count > 50) && <option value="50">50</option> }
+          <option value="20">20</option>
+          {(this.props.count > 20) && <option value="30">30</option> }
```
Comment (amardesi) [inline app/jsx/components/SortComp.jsx]: i.e. if there are 24 pages, you should be able to select 30 rows so you can see them all

File: app/jsx/components/ScholWorksComp.jsx (hunk 3 above)
Comment (martinhaye) [inline app/jsx/components/ScholWorksComp.jsx]: Yay! Thank you for including the thumbnails Andy.

File: app/jsx/components/ScholWorksComp.jsx (hunk 1 above)
Comment (martinhaye) [inline app/jsx/components/ScholWorksComp.jsx]: No problem.

File: app/jsx/components/ScholWorksComp.jsx (hunk 2 above)
Comment (martinhaye) [inline app/jsx/components/ScholWorksComp.jsx]: Is this the 'et al' link that shows up on the search page when a searched-for item has lots of authors? That one isn't currently working for me. It shows up as "et al" but just refreshes the page when I click it. Not a merge-blocker, just fyi.

File: app/jsx/components/SortComp.jsx (hunk 4 above)
Comment (martinhaye) [inline app/jsx/components/SortComp.jsx]: Oh, right! Cool.

File: app/jsx/pages/SearchPage.jsx [hunk 5]
```diff
@@ -77,8 +77,8 @@ class FacetItem extends React.Component {
   }
 
   handleChange = event => {  // compact syntax that takes care of binding the event handler
-    var filter
-    var filter_cleanup = []
+    let filter
+    let filter_cleanup = []
```
Comment (martinhaye) [inline app/jsx/pages/SearchPage.jsx]: Yep, they say 'let' is the new 'var'. I like...

File: app/jsx/components/ScholWorksComp.jsx (hunk 2 above)
Comment (amardesi) [inline app/jsx/components/ScholWorksComp.jsx]: This was fixed in my latest Pull Request, FYI.

**Pull request merged**